app/forms/declaration/DeclarationAdditionalActors.scala

```diff
@@ -28,6 +29,15 @@ case class DeclarationAdditionalActors(eori: Option[String], partyType: Option[S
 
   def toJson: JsValue = Json.toJson(this)(DeclarationAdditionalActors.format)
 
+  def extractPartyType: String =
```

Comment:
Sounds like a method that should belong on  `PartyType`

e.g. `PartyType.toString`

Reply:
Then you can  do `additionalActors.partyType.map(_.toString).getOrElse("")`

Reply:
Good point, will move it

Reply:
Generaly `toString` in domain code is not good thing unless is stable and fully defined - for enum 'https://www.scala-lang.org/api/current/scala/Enumeration.html' we don't have this case

Reply:
Moved to the messages, which are based on the code